config/locales/client.en.yml

```
@@ -6243,6 +6243,9 @@ en:
           reset_confirm: "Are you sure you want to reset to the default %{fieldName} and lose all your changes?"
           save_error_with_reason: "Your changes were not saved. %{error}"
           instructions: "Customize the template in which all html emails are rendered, and style using CSS."
+        font_style:
+          title: "Font Style"
```

```suggestion
          title: "Font style"
```

Per https://meta.discourse.org/t/formatting-text-in-discourse-documentation-and-uis/324637